clang/lib/AST/ExprConstant.cpp

```diff
@@ -2805,7 +2805,7 @@ static bool handleIntIntBinOp(EvalInfo &Info, const Expr *E, const APSInt &LHS,
     unsigned SA = (unsigned) RHS.getLimitedValue(LHS.getBitWidth()-1);
     if (SA != RHS) {
       Info.CCEDiag(E, diag::note_constexpr_large_shift)
-        << RHS << E->getType() << LHS.getBitWidth();
+          << RHS << E->getType() << LHS.getBitWidth();
```

Comment:
Only unrelated whitespace changes in the file, so I think the whole file can be reverted.

Reply:
Reverted changes. Just to mention that 'git clang-format' proposed spaces in this comment and comments below.

Reply:
FWIW, you should be using `clang-format-diff` so that you're only formatting the changes made in the PR: https://clang.llvm.org/docs/ClangFormat.html#script-for-patch-reformatting